OfficialTutorials/SpaceShuttle.json

```diff
@@ -68,7 +75,8 @@
           "es": "Pasa por 2000ft",
           "zh": "下降通过2000英尺",
           "bg": "Пресечете 2000фт",
-          "tr": "2000ft'den geç"
+          "tr": "2000ft'den geç",
+          "az": "2000ft'den geç"
```

Comment:
This translation needs to be rewritten, apostrophes are not used to separate suffixes in Azerbaijani language